README.md

```diff
@@ -14,54 +23,88 @@ For documentation on previous versions of the Molecular Targets Platform, please
 
 #### Contents
 - [Datasets](#datasets)
-- [Somatic Alterations](#Somatic_Alterations)
-- [Gene Expression](#Gene_Expression)
+- [DNA Sequencing](#DNA_Sequencing)
+- [RNA Sequencing](#RNA_Sequencing)
+- [Methylation](#Methylation)
 - [Data Availability](#Data_Availability)
 
 ## Datasets
 
-While adult pan-cancer repositories have existed and accelerated cancer research for a decade, pediatrics cancers have been excluded, despite having different genetic and molecular etiologies than adult cancers. Over the past few years larger pediatric consortia, both disease-specific and pan-cancer, have tried to address this disparity. The Molecular Targets Platform is harmonizing the data from across these different consortia in one unified location where it can be queried for associations between putative targets and pediatric cancers. As the project is ongoing, more data will continue to be added, but this current release includes 3 pediatric consortia datasets as well as GTEx data for comparisons to normal tissue expression:
+While adult pan-cancer repositories have existed and accelerated cancer research for a decade, pediatrics cancers have been excluded, despite having different genetic and molecular etiologies than adult cancers. 
+Over the past few years larger pediatric consortia, both disease-specific and pan-cancer, have tried to address this disparity. 
+The Molecular Targets Platform is harmonizing the data from across these different consortia in one unified location where it can be queried for associations between putative targets and pediatric cancers. 
+As the project is ongoing, more data will continue to be added, but this current release includes **XXX** pediatric consortia datasets as well as GTEx data for comparisons to normal tissue expression:
 
 | Dataset | Number of Samples |
 | --- | --- |
-| PBTA | 2,984 |
-| TARGET | 4,489 |
-| GMKF | 884 |
+| PBTA | 4,164 |
+| TARGET | 5,252 |
+| GMKF | 881 |
+| DGD | 1,799 |
```

Comment:
```suggestion
| DGD (CHOP P30 panel) | 1,799 |
```

Forgot we made this change in the tables!

Reply:
Ugh right good catch. Now I need to update the cohort descriptions